Added client-side caching for assets with version query string. (#64)
netzstrategen/wordpress-core-standards: README.txt

  virtual host or the whole server (the latter is not recommended for production
  servers).

>   Adds long client-side caching time for assets with "ver" query string via .htacces.

Reviewer: ```suggestion
- Adds client-side caching for assets having a `'ver'` query string.
```
